third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -59,54 +64,15 @@ resource "google_container_node_pool" "primary_preemptible_nodes" {
     metadata = {
       disable-legacy-endpoints = "true"
     }
-
-    oauth_scopes = [
+    # Google now recommends custom service accounts that have cloud-platform scope and permissions granted via IAM Roles.
+    service_account = google_service_account.default.email
+    oauth_scopes    = [
       "https://www.googleapis.com/auth/cloud-platform"
     ]
   }
 }
 ```
 
-## Example Usage - with the default node pool
```

Comment:
drive-by: This + the node pool example being removed are intentional and should be kept, even if they don't fit the best practices for configuring the resource. While we'd prefer users configure resources using the recommended setups, I'd rather we provide clear documentation on what alternatives look like vs forcing users to experiment themselves.

Reply:
I'll revert this change for both resources.

Reply:
The GKE resource is such a mess. There are so many open breaking change issues for it.

Reply:
It's a busy resource!